background.js

```diff
@@ -349,6 +373,10 @@ function shouldContainInto (url, tab) {
     handleUrl = false;
   }
 
+  if (!handleUrl && extensionSettings.allowlist.length!=0 && isAllowlistedURL(url)) {
```

Comment:
Maybe we should check this before the others, otherwise !handleUrl will always be true and we will be checking unnecessarily.

Reply:
Yes, good point 👍🏻

Reply:
mmmm I agree, but i've just follow the already code created by whitelist feature. If we don't want to check unnecessarily, I propose also to put the "whitelist if" before the others as well.

Reply:
Yea, sure, do that.

Reply:
Ok, now the code check if is `allowedHost` on top, as same level of `isGoogleURL()` and move before checks the "whitelist if".
I change the order and description on preferences, reflecting the real processing even it would be strange if someone put the domain on `allowlist` and `whitelist`